GTBitsOfGood/southface
# Nabeel/126 shopping cart

## Making shopping cart view work

Issue Number(s): #126 

This PR makes the shopping cart view work as described in the Figma. 

### Checklist

-  onClick of the tab it opens the shopping cart
 - Shopping Cart is deleted from the NavBar
 - active report data is shown in the shopping cart
 - removing card in shopping cart reflects active report


### How to Test

- Add things to report
- Click on button
- Observe that shopping cart view shows everything in the report
- Remove a standard by clicking "remove" on a standard
- Go to report builder page
- Observe that the standard is removed

### Comments
Right now, because #133 is still being worked on, the title of the report is just a dummy title. This should be addressed after that issue is closed. 

Also, the button to open and the button to close the view are actually two different buttons that happen to be placed on top of one another. Let me know if that would be problematic, but this is the easiest way to implement the figma design.
